docs/docs/get-started.mdx

```diff
@@ -1,107 +0,0 @@
----
```

Comment:
Instead of removing it how about adding some getting started guides? :) Then we don't need to add a redirect!

Reply:
I can add a first verrsion to this PR, what do you think about this for content:

```
Community
+
Kratos etc. Quickstarts
+ 
Guides Overview: (Cloud/Kratos/Hydra/Keto/Oathk)
+
ExpressJS / NodeJS
https://www.ory.sh/docs/start-building/server-side-web-app#expressjs--nodejs
Next.js / React
https://www.ory.sh/login-spa-react-nextjs-authentication-example-api-open-source/
https://www.ory.sh/nextjs-authentication-spa-custom-flows-open-source/
OAuth 2.0 Server
https://www.ory.sh/run-oauth2-server-open-source-api-security/
Ory Cloud Projects
https://www.ory.sh/docs/concepts/project
Ory Cloud Services & APIs
https://www.ory.sh/docs/concepts/services-api
Ory Cloud Developer Plan
https://www.ory.sh/docs/developer
+
Concept Docs Intro
+
API references
```

Reply:
yes, that would be great! we definitely need to make this beautiful in the next few days as it is the second document people will look at